tsc/process/release.md

```diff
@@ -0,0 +1,20 @@
+**Release Process for OpenVDB**
+
+The following assumes that the current OpenVDB library version number is 6.0.0 and the new version number is 6.1.0. Adjust for the actual version numbers as appropriate.
+
+- [ ] Open a Jira "Release OpenVDB 6.1.0" ticket with "OpenVDB_6.1.0" as the Fix Version.
+- [ ] Update `openvdb/CHANGES` and `openvdb/doc/changes.txt` with release notes.
+- [ ] Update `openvdb/INSTALL` with the library version number and any directory hierarchy changes.
+- [ ] Push the above changes to `openvdb/master`, associating them with the Jira ticket created earlier, and verify that the Travis CI build runs successfully.
+- [ ] Draft a new [GitHub release](https://github.com/AcademySoftwareFoundation/openvdb/releases). Title it "OpenVDB 6.1.0" and tag it as `v6.1.0`.
+
+- [ ] Update `openvdb-website/contents/download/index.html` with the new version number (6.1.0) and release date, and `openvdb-website/contents/index.html` with a news item. Open the latter in a browser and check that the website renders correctly and that there are no broken links.
+- [ ] Build the documentation (both the `doc` and `pydoc` targets) and replace the contents of `openvdb-website/contents/documentation/doxygen/` with the output.  [_This step should be automated, and the thousands of files it generates should preferably not be committed to the repository._]
+- [ ] Push the above changes to `openvdb-website/master`, associating them with the Jira ticket created earlier.
+- [ ] Post a release announcement to the [OpenVDB forum](https://groups.google.com/forum/#!forum/openvdb-forum).
+
+- [ ] In preparation for the next release, change one or more of `OPENVDB_LIBRARY_PATCH_VERSION_NUMBER`, `OPENVDB_LIBRARY_MINOR_VERSION_NUMBER` and `OPENVDB_LIBRARY_MAJOR_VERSION_NUMBER` in `openvdb/version.h`.  Unless it is known that the next release will include API- or ABI-breaking changes, increment only the patch number to begin with (in this case, from 6.1.0 to 6.1.1).  In `openvdb/doxygen-config` update `PROJECT_NUMBER`, `OPENVDB_VERSION_NAME`, `OPENVDB_ABI_VERSION_NUMBER` and the `@vdbnamespace` alias to match `version.h`.  Push these changes to `openvdb/master`.
```

Comment:
Perhaps a missing step to add the "In Development" for the next version in openvdb/changes?

(As above, no direct pushes to master, so this would still need to be a PR that's approved.)

Reply:
I've added that to the "In preparation" step.